### ISSUE TITLE
Training process and loss understanding

### ISSUE THREAD

**Issue by SanketDhuri** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#1464]
1	How does training process works and why do get such loss?
We have a dataset of min 1000 audios/speaker of duration >= 2sec but still on training on this dataset for 20/50/100/200 epochs it doesn’t reach satisfactory loss 
eg last observed loss is :
loss_disc=2.583, loss_gen=3.318, loss_fm=13.223,loss_mel=24.893, loss_kl=1.822 
 
this both losses must be nearly equal and less which doesn’t happen though it gives output good enough for voice conversion 

2	What is the importance of Index file during inference ?
3	Can we change accent with voice in RVC ? consider model is finetuned to voice of US accent and for testing we are passing audio of Indian accent to rvc what output should we aspect ?

**Comment by trivikramak** [comment]
Thanks to someone who explained it in this thread - https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/issues/507#issuecomment-1586986107

**Comment by SanketDhuri** [comment]
and about this questions
1 What is the importance of Index file during inference ?
2 Can we change accent with voice in RVC ? consider model is finetuned to voice of US accent and for testing we are passing audio of Indian accent to rvc what output should we aspect ?
